development/verify-locale-strings.js

```diff
@@ -192,7 +192,7 @@ async function verifyEnglishLocale() {
       'app/scripts/constants/**/*.js',
       'app/scripts/constants/**/*.ts',
       'app/scripts/platforms/**/*.js',
-      'app/scripts/controllers/push-platform-notifications/utils/get-notification-message.ts',
+      'app/scripts/controllers/**/*.ts',
```

Comment:
Updating this setting as this file does not exist.

We still have translation strings inside the `/controllers` folder (e.g. the translations for push notifications, this unfortunately cannot be shared cross platform).